applications/ConvectionDiffusionApplication/custom_elements/symbolic_d_convection_diffusion_explicit.h

```diff
@@ -0,0 +1,317 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main author:     Riccardo Tosi
+//
+
+#ifndef KRATOS_SYMBOLIC_D_CONVECTION_DIFFUSION_EXPLICIT_H
+#define KRATOS_SYMBOLIC_D_CONVECTION_DIFFUSION_EXPLICIT_H
+
+// System includes
+
+
+// External includes
+
+
+// Project includes
+#include "symbolic_qs_convection_diffusion_explicit.h"
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/**
+ * @class SymbolicDConvectionDiffusionExplicit
+ * @ingroup ConvectionDiffusionApplication
+ * @brief This element solves the convection-diffusion equation, stabilized with
+ * algebraic subgrid scale or orthogonal subgrid scale.
+ * @details This element solves the convection-diffusion equation:
+ * $ \frac{\partial \phi}{\partial t} + v \cdot  \nabla \phi + \phi \nabla \cdot v - \nabla \cdot k \nabla \phi = f $
+ * where $ \phi $ is the scalar unknown, $ v $ the convective velocity, $ k > 0 $ the diffusivity coefficient,
+ * $ f $ the forcing term.
+ * Dynamic algebraic subgrid scale and dynamic orthogonal subgrid scale methods are exploited for stabilization.
+ * The element is designed to use an explicit integration method.
+ * @author Riccardo Tosi
+ */
+template< unsigned int TDim, unsigned int TNumNodes = TDim + 1>
+class SymbolicDConvectionDiffusionExplicit : public SymbolicQSConvectionDiffusionExplicit<TDim,TNumNodes>
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+        typedef SymbolicQSConvectionDiffusionExplicit<TDim,TNumNodes> BaseType;
+        typedef typename BaseType::ElementVariables ElementVariables;
+        typedef Node < 3 > NodeType;
+        typedef Geometry<NodeType> GeometryType;
+        typedef Geometry<NodeType>::PointsArrayType NodesArrayType;
+        typedef Vector VectorType;
+        typedef Matrix MatrixType;
+        typedef std::size_t IndexType;
+        typedef std::vector<std::size_t> EquationIdVectorType;
+        typedef std::vector< Dof<double>::Pointer > DofsVectorType;
+        typedef GeometryData::IntegrationMethod IntegrationMethod;
+
+    /// Pointer definition of SymbolicDConvectionDiffusionExplicit
+    KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(SymbolicDConvectionDiffusionExplicit);
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    //Constructors.
+
+    /// Default constuctor.
+    SymbolicDConvectionDiffusionExplicit(IndexType NewId, GeometryType::Pointer pGeometry);
+    SymbolicDConvectionDiffusionExplicit(IndexType NewId, GeometryType::Pointer pGeometry, Properties::Pointer pProperties);
```

Comment:
```suggestion
    SymbolicDConvectionDiffusionExplicit(
        IndexType NewId,
        GeometryType::Pointer pGeometry);
    
    SymbolicDConvectionDiffusionExplicit(
        IndexType NewId,
        GeometryType::Pointer pGeometry,
        Properties::Pointer pProperties);
```
Style convention.

Reply:
And move to the `constructors` section of the Doxygen (these are not the default ones).

Reply:
Ok, done.